src/test/java/core/basesyntax/service/operations/BalanceOperationTest.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class BalanceOperationTest {
+    private FruitTransaction transaction;
+
+    @Test

```

Comment:
Test method name is not descriptive. Consider following the convention of naming test methods in a way that describes its purpose or the functionality it is testing.